cmd/samInfo/readLength_test.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"github.com/vertgenlab/gonomics/fileio"
+	"testing"
+)
+
+var ReadLengthTests = []struct {
+	InFile       string
+	OutFile      string
+	ExpectedFile string
+}{
+	{InFile: "testdata/readLength/small.sam",
+		OutFile:      "testdata/readLength/test.readLength.txt",
```

Comment:
Could you try to tests a few of these function directly instead of creating additional expected test files?
For example, you could write  a test called
```go
func TestThresholdCalc(threshold, covHistogram, totalCount) {
    expected := 10 // or any manual value
    number := ThresholdCalc()
    if number != expected {
        t.log("error with ThresholdCalc()")   
    }
}
`
and assert for equality by creating 1 (or more) values. Writing more "meaningful" tests will reduce the number of in line annotations from CodeCov and make future refactoring efforts more seemless if all we have to do is change test values father than an arrary of expected test files